packages/terra-html-table/src/Table.module.scss

```diff
@@ -0,0 +1,125 @@
+/* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+:local {
+  .table {
+    background-color: var(--terra-html-table-background-color, rgba(255, 255, 255, 0));
+    border: var(--terra-html-table-border, 0);
+    border-collapse: collapse;
+    border-spacing: 0;
+    max-width: 100%;
+    width: 100%;
+
+    caption {
+      padding-bottom: 0.5em;
+    }
+
+    tr {
+      display: table-row;
+      margin-bottom: 0;
+    }
+
+    td {
+      border: 0;
+      border-left: var(--terra-html-table-cell-border-left, 0);
+      display: table-cell;
+      text-align: left;
+      vertical-align: top;
+
+      &:first-of-type {
+        border-left: var(--terra-html-table-first-cell-border-left, 1px solid #c8cacb);
+      }
+
+      &:last-of-type {
+        border-right: var(--terra-html-table-last-cell-border-right, 1px solid #c8cacb);
+      }
+    }
+
+    th {
+      border: 0;
+      border-left: var(--terra-html-table-header-cell-border-left, 0);
+      display: table-cell;
+      text-align: left;
+      vertical-align: top;
+
+      &:first-of-type {
+        border-left: var(--terra-html-table-first-header-cell-border-left, 1px solid #c8cacb);
+      }
+
+      &:last-of-type {
+        border-right: var(--terra-html-table-last-header-cell-border-right, 1px solid #c8cacb);
+      }
+    }
+
+    thead {
+      background-color: var(--terra-html-table-thead-background-color, #dedfe0);
+      display: table-header-group;
+
+      tr {
+        border-bottom: var(--terra-html-table-thead-tr-border-bottom, 0);
```

Comment:
Should `--terra-html-table-thead-tr-border-bottom,` be `table-row` instead of `tr` and below, should `--terra-html-table-thead-th-color` be `table-head` instead of `th` to follow the pattern used above for `th` and `td` styles